tests/legacy/test_task.py

```diff
@@ -11,11 +11,12 @@ class TestTask:
     These validate duplicate and partially constructed tasks.
     """
 
-    def test_task_duplicate(self):
+    @pytest.mark.library_only
+    def test_task_duplicate(self, init_kwargs):
         """Test to validate duplicate and partially constructed tasks."""
-        with Task("task") as t:  # noqa: F841
+        with Task("task", **init_kwargs) as t:  # noqa: F841
             with pytest.raises(DaqError) as dupe_exception:
-                u = Task("task")
+                u = Task("task", **init_kwargs)
```

Comment:
This reminds me, we should write tests for `grpc_options.initialization_behavior` (but not in this PR).

Reply:
Sure Brad, This will taken up in [Technical Debt 2395680](https://dev.azure.com/ni/DevCentral/_workitems/edit/2395680): Add test case for the task initialization behavior in gRPC interpreter